server/metrics/metrics.go

```diff
@@ -143,6 +144,10 @@ func InitializeMetrics() func() {
 		initializeQuotaScopes()
 
 		SchemaMetrics = root.SubScope("schema")
+
+		if config.DefaultConfig.GlobalStatus.Enabled {
+			GlobalSt = NewGlobalStatus()
```

Comment:
should we always create it so that caller doesn't rely on this config flag? You can move this check inside the expose functions of global status to not do anything in case this is disabled.

Reply:
I changed that too.